pytorch_lightning/loggers/tensorboard.py

```diff
@@ -46,10 +46,13 @@ class TensorBoardLogger(LightningLoggerBase):
     preinstalled.
 
     Example:
-        >>> from pytorch_lightning import Trainer
-        >>> from pytorch_lightning.loggers import TensorBoardLogger
-        >>> logger = TensorBoardLogger("tb_logs", name="my_model")
-        >>> trainer = Trainer(logger=logger)
+
+    .. code-block:: python
+
```

Comment:
I believe this is not necessary, `Example:` renders already

but haven't checked

```suggestion
```

Reply:
Yes but I thought I'd keep it consistent with other loggers

Reply:
well, the difference is in `:` and `::`
with `:` it is recognized as a section title
with `::` it is recognized as caption followed by code
https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/Codeblocks.html